test/Libraries/Microsoft.Extensions.Compliance.Redaction.Tests/ErasingRedactorTests.cs

```diff
@@ -1,60 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System;
-using Xunit;
-
-namespace Microsoft.Extensions.Compliance.Redaction.Test;
-
-public class ErasingRedactorTests
```

Comment:
I don't see `ErasingRedactor` to be removed in that PR

Reply:
Whoops, I deleted it at some point and then brought it back. I restored the test.